src/components/AlertBar/AlertBar.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
+import { connect } from 'react-redux'
+import { AlertBar, AlertStack } from '@dhis2/ui'
+import PropTypes from 'prop-types'
+
+import { sGetAlertMessage } from '../../reducers/alert'
+import { acClearAlertMessage } from '../../actions/alert'
+
+export const Alert = ({ message, onClose }) =>
+    message ? (
+        <AlertStack>
+            <AlertBar onHidden={onClose} permanent>
```

Comment:
For DV I used the `variant` prop for different types of alerts, and the `permanent` prop is only set for errors or warnings.
Ie. you would want the loading message to disappear automatically. That is built in so you don't have to handle the timeout in the app.
See: https://github.com/dhis2/data-visualizer-app/blob/master/packages/app/src/components/Snackbar/Snackbar.js#L16-L37

Reply:
The way the alertbar currently works is that the message stays until  the app detects that the dashboard is finished loading. So I actually want the message to be permanent in this case. When the loading is complete, then the message in redux is cleared and the Alert renders null.

Reply:
ok, sounds good! 👍